### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by iwillteixeira**
1. Initialize React App
2. Add Redux
3. Add structure
4. Add components

### CODE REVIEW

File: src/components/AddBooks/AddBooks.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      <input type="text" name="" id="" />
+      <select defaultValue="">
+        <option value="" disabled>Category</option>
+        {categories.map((option) => (
+          <option key={option.id} value={option.categories}>{option.categories}</option>
+        ))}
+      </select>
+    </div>
+  );
+};
```
Comment (Mihndim2020) [inline src/components/AddBooks/AddBooks.js]: - [x] As it is a requirement for this project, please add a `CreateNewBook` component that; 
- Renders a form with inputs for the title and the author of the book
- Has a button to submit(the button wouldn't add the book yet, this is just a placeholder for now)

File: src/components/BookList/BookList.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            <span>CURRENT CHAPTER</span>
+            <p>Introduction</p>
+            <button type="button">UPDATE PROCESS</button>
+          </div>
+        </div>
+      ))}
+    </div>
+  );
+};
+
```
Comment (Mihndim2020) [inline src/components/BookList/BookList.js:16]: - [x]  As it is a requirement for this project, please add a `Book` component that; 
- Takes the book data as a prop and renders the title and the author
- Has a button for removing books(the button wouldn't remove the books yet, this is just a placeholder for now)
Please checkout the screenshot below for clarifications. 
booklist

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages,
:heavy_check_mark: Descriptive Readme file,
:heavy_check_mark: No `Linter` errors.

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/AddBooks/AddBooks.js (hunk 1 above)
Comment (iwillteixeira) [reply src/components/AddBooks/AddBooks.js]: Add submit button, title, and author input.

File: src/components/BookList/BookList.js (hunk 2 above)
Comment (iwillteixeira) [reply src/components/BookList/BookList.js:16]: It already has the remove option rendered, and the book and booklist already was rendering title and author.

**Review by D4nielJ — APPROVED** [review]
## Hello @iwillteixeira,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**